python/tskit/tables.py

```diff
@@ -64,11 +64,15 @@ class NOTSET(metaclass=NotSetMeta):
 @metadata.lazy_decode
 @dataclass(**dataclass_options)
 class IndividualTableRow(util.Dataclass):
+    """
+    A row in an :class:`IndividualTable`.
+    """
+
     __slots__ = ["flags", "location", "parents", "metadata"]
-    flags: int
-    location: np.ndarray
-    parents: np.ndarray
-    metadata: Optional[Union[bytes, dict]]
+    flags: int  #: See :attr:`Individual.flags`
```

Comment:
Agree there's no point in duplicating here.

Might was well use the standard docstring form though, like
```
""" See :attr:`Individual.flags` """
flags: int
```
as there will be cases in which the form you have will be too long for a single line, and it's good to keep things uniform, even if just for greppability.